### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by chiaradiamarcelo**
Release 1.0.0

### CODE REVIEW

File: src/test/java/com/exasol/adapter/dialects/postgresql/PostgreSQLSqlDialectIT.java [hunk 1]
```diff
@@ -38,7 +38,7 @@
     private static final String POSTGRES_CONTAINER_NAME = "postgres:9.6.2";
     private static final String DOCKER_IP_ADDRESS = "172.17.0.1";
     private static final int POSTGRES_PORT = 5432;
-    private static final String VIRTUAL_SCHEMAS_JAR_NAME_AND_VERSION = "virtual-schema-dist-7.0.0-postgresql-0.1.0.jar";
+    private static final String VIRTUAL_SCHEMAS_JAR_NAME_AND_VERSION = "virtual-schema-dist-8.0.0-postgresql-0.1.0.jar";
```
Comment (AnastasiiaSergienko) [inline src/test/java/com/exasol/adapter/dialects/postgresql/PostgreSQLSqlDialectIT.java:41]: I think the name here should end with `postgresql-1.0.0.jar"`, shouldn't it?

File: src/test/java/com/exasol/adapter/dialects/postgresql/PostgreSQLSqlDialectTest.java [hunk 2]
```diff
@@ -44,14 +44,12 @@ void beforeEach() {
 
     @Test
     void testCreateRemoteMetadataReader() {
-        assertThat(getMethodReturnViaReflection(this.dialect, "createRemoteMetadataReader"),
```
Comment (AnastasiiaSergienko) [inline src/test/java/com/exasol/adapter/dialects/postgresql/PostgreSQLSqlDialectTest.java:47]: Can you remove the class that contains getMethodReturnViaReflection now?

File: doc/dialects/postgresql.md [hunk 3]
```diff
@@ -25,8 +25,8 @@ The SQL statement below creates the adapter script, defines the Java class that
 ```sql
 CREATE OR REPLACE JAVA ADAPTER SCRIPT ADAPTER.JDBC_ADAPTER AS
   %scriptclass com.exasol.adapter.RequestDispatcher;
-  %jar /buckets/<BFS service>/<bucket>/virtual-schema-dist-7.0.0-postgresql-0.1.0.jar;
-  %jar /buckets/<BFS service>/<bucket>/postgresql-<version>.jar;
+  %jar /buckets/<BFS service>/<bucket>/virtual-schema-dist-8.0.0-postgresql-0.1.0.jar;
```
Comment (AnastasiiaSergienko) [inline doc/dialects/postgresql.md:28]: Oh. here is also an incorrect jar name.

**Review by AnastasiiaSergienko — CHANGES_REQUESTED** [review]

**Review by AnastasiiaSergienko — APPROVED** [review]

**Review by redcatbear — APPROVED** [review]
Approved without findings. Good job.

**Pull request merged**